build/use-lodash-es.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function useLodashEs() {
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        const source = path.node.source;
```

Comment:
I'm not sure. Its a fairly small piece of functionality isolated from the majority of the code base. I personally wouldn't spend too much time on worrying about aesthetic changes.

But if you have a really strong opinion here, I can attempt to make the change.